docs/source/en/using-diffusers/kerascv.mdx

```diff
@@ -10,170 +10,108 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# Using KerasCV Stable Diffusion Checkpoints in Diffusers
+# Load different Stable Diffusion formats
 
-<Tip warning={true}>
+Stable Diffusion models are available in different formats depending on the framework they're trained and saved with and where you download them from. Converting these formats for use in 🤗 Diffusers allows you to use all the features supported by the library, such as [using different schedulers](schedulers) for inference or [building your custom pipeline](write_own_pipeline).
```

Comment:
I would maybe also highlight the speed optimization that we offer from the library too as that is pretty convenient for practitioners.